plugins/destination/postgresql/main.go

```diff
@@ -1,17 +1,20 @@
 package main
 
 import (
+	"context"
+
 	"github.com/cloudquery/cloudquery/plugins/destination/postgresql/client"
 	"github.com/cloudquery/cloudquery/plugins/destination/postgresql/resources/plugin"
-	"github.com/cloudquery/plugin-sdk/v3/plugins/destination"
-	"github.com/cloudquery/plugin-sdk/v3/serve"
+	pluginSDK "github.com/cloudquery/plugin-sdk/v4/plugin"
+	"github.com/cloudquery/plugin-sdk/v4/serve"
 )
 
 const (
 	sentryDSN = "https://19d1257d36854a51b17c06614e76dc2d@o1396617.ingest.sentry.io/4503896817336320"
 )
 
 func main() {
-	p := destination.NewPlugin("postgresql", plugin.Version, client.New, destination.WithDefaultBatchSize(1000))
-	serve.Destination(p, serve.WithDestinationSentryDSN(sentryDSN))
+	p := pluginSDK.NewPlugin("postgresql", plugin.Version, client.New)
+	server := serve.Plugin(p, serve.WithPluginSentryDSN(sentryDSN))
+	server.Serve(context.Background())
```

Comment:
We seem to be ignoring `error` result from Serve which gets caught in the linter.